rtf_vis_tool/src/App.css

```diff
@@ -0,0 +1,22 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+#root {
+  width: 100%;
+  height: 100%;
+  margin: 0px;
+  padding: 10px;
+}
+
+body {
+  background: white;
+  position: fixed;
+}
+
+.scale-range-container {
+  display: flex;
+  flex-direction: row;
+}
```

Comment:
some basic styling added to App.js

Reply:
Cool. Could you add some brief comments explaining the choice of padding/margins/ which parts of the project they'll apply to?